### DISCUSSION TITLE
匹配的表情居然动态更新了

### DISCUSSION THREAD

**Discussion by xyl23333** [Richasy/Bili.Uwp#956] [常规]
显示的表情可以动态更新到愚人节版本。请问是两个版本的表情字符有同样的url，还是b站开放了表情符号的对照文件？我还是想了解一下。

![2022-04-01 _5_.png](https://s2.loli.net/2022/04/01/4EF7Xn26BQao3lW.png)

**Comment by Richasy** [comment]
B站返回的数据里包含表情链接，应用只是按照链接显示图片
